src/lib/utils/options/readers/tsconfig.ts

```diff
@@ -40,21 +40,32 @@ export class TSConfigReader extends OptionsComponent {
             return;
         }
 
+        let file: string | undefined;
+
         if (TSConfigReader.OPTIONS_KEY in event.data) {
-            this.load(event, Path.resolve(event.data[TSConfigReader.OPTIONS_KEY]));
+            const tsconfig = event.data[TSConfigReader.OPTIONS_KEY];
+
+            if (/tsconfig\.json$/.test(tsconfig)) {
```

Comment:
I think that this line is responsible for making typedoc 0.14 a breaking change for my projects.

The path that I use for my config file is something like `node_modules/my-config-project/config/typedoc.config.json`

This worked fine in 0.13 but breaks in 0.14.

I now need to update the path on 60+ projects..